shell/egl_window.cc

```diff
@@ -336,6 +336,9 @@ void EglWindow::redraw(void* data,
   if (callback)
     wl_callback_destroy(callback);
 
+  window->m_callback = wl_surface_frame(window->m_surface);
+  wl_callback_add_listener(window->m_callback, &frame_listener, window);
```

Comment:
I think this belongs in a different patch. Obviously not adding a new frame callback would not trigger it a new one. So this seems like a small bug missing adding it. Right?

Reply:
this is for rendering the FPS overlay window